lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1540,6 +1608,36 @@ def test_import_val_data(self):
         self.assertEqual(video_data['encoded_videos'][0]['url'], 'http://example.com/video')
         self.assertEqual(video_data['encoded_videos'][0]['file_size'], 222)
         self.assertEqual(video_data['encoded_videos'][0]['bitrate'], 333)
+        # verify transcript data
+        video_transcript = get_video_transcript(video.edx_video_id, 'ar')
+        for key, value in self.get_video_transcript_data('test_edx_video_id').items():
```

Comment:
its good practice to use `self.get_video_transcript_data('test_edx_video_id').iteritems()`